src/Application.js

```diff
@@ -33,20 +31,18 @@ export class Application {
    * @param {Boolean} [config.features.sfxVolume] A boolean denoting that this game has adjustable sound effects volume in it
    */
   constructor({ features, hintPlayer = new HintSequencePlayer() } = {}) {
-    /**
-     * @member {StateManager} The state manager for this application instance. Maintains subscribable properties for
-     *                        whether or not audio is muted, captions are displayed, or the game is paused.
-     */
-    this.state = new StateManager();
-    this.state.addField(ready, false);
-    this.state.addField(pause, false);
-    this.state.addField(captionsMuted, true);
-    this.state.addField(playOptions, {});
-
-    this.state.addField(soundVolume, 1);
-    this.state.addField(musicVolume, 1);
-    this.state.addField(voVolume, 1);
-    this.state.addField(sfxVolume, 1);
+    this.state = {
+      ready: new Property(false),
+      pause: new Property(false),
+      captionsMuted: new Property(true),
+      playOptions: new Property({}),
+      soundVolume: new Property(1),
+      musicVolume: new Property(1),
+      voVolume: new Property(1),
+      sfxVolume: new Property(1)
+    };
+
+    this.state.ready;
```

Comment:
huh?

Reply:
Something I forgot to remove. I'll open another pr to remove it.